### PR TITLE
Add support for sending a commit status to github if the contributor has signed CLA.

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

**Comment by boennemann** [comment]
Thanks for your contribution @ketan.
I'm not actively maintaining this anymore, and unfortunately don't have the time to review this anytime soon. @lieutdan13 Can you review and merge this?

File: src/lib/answerPullRequest.coffee [hunk 1]
```diff
@@ -20,12 +23,36 @@ exports = module.exports = (req, res, options, contractors, payload) ->
...
+
     comment.send options.token, commentData, (err, data) ->
       if err
+        success = false
         console.log err
-        console.log   'Fatal Error: GitHub refused to comment'
-        res.send 500, 'Fatal Error: GitHub refused to comment'
+        console.log   "Fatal Error: GitHub refused to comment"
       else
         href = payload.pull_request._links.html.href
```
Comment (lieutdan13) [inline src/lib/answerPullRequest.coffee:40]: You set href at the top. I don't think this is needed.

**Comment by lieutdan13** [comment]
Thanks for the PR @ketan. 2 things:
1) How does one trigger this event and what does this do when you trigger it? (I'm sorry. This is a project I haven't visited in awhile).
2) Please write a unit test (or a few) to accompany your code.

**Comment by lieutdan13** [comment]
Closing due to > 1 year and no feedback from the OP.

**Pull request closed without merging**